open-reac/src/main/java/com/powsybl/openreac/parameters/output/OpenReacResult.java

```diff
@@ -97,8 +114,61 @@ public List<NetworkModification> getAllModifs() {
     }
 
     public void applyAllModifications(Network network) {
-        for (NetworkModification modif : getAllModifs()) {
+        for (NetworkModification modif : getAllNetworkModifications()) {
             modif.apply(network);
         }
+
+        // update target of ratio tap changers specified as variable by user
+        getTapPositionModifications().stream()
```

Comment:
🥇 Your code is unreadable even if it is maybe very right :-) Many improvements since your first PR! If the target V is NaN, I think you have to change it too.

Reply:
Same for shunts!